react-native-libraries.json

```
@@ -9054,6 +9054,13 @@
     "web": true,
     "expo": true
   },
+  {
+    "githubUrl": "https://github.com/ArunGovil/react-native-flatboard",
+    "images": ["https://raw.githubusercontent.com/ArunGovil/react-native-flatboard/main/src/assets/demo.webp"],
```

```suggestion
    "images": ["https://raw.githubusercontent.com/ArunGovil/react-native-flatboard/main/src/assets/cover.webp"],
```

It looks like `demo` file no longer exist.

---

Updated.